user/managing-os/minimal-templates.md

```diff
@@ -162,9 +166,31 @@ Documentation on all of these can be found in the [docs](/doc)
 You could, of course, use qubes-vm-recommended to automatically install many of these, but in that case you are well on the way to a standard Debian template.
 
 
+### CentOS
+
+As is the case with above-mentioned Minimal Templates, the required packages are to be installed in the running template with the following command (replace `packages` with a space-delimited list of packages to be installed):
+
+    [user@your-new-clone ~]$ sudo yum install packages
+
+Use case | Description | Required steps
+--- | --- | ---
+**Standard utilities** | If you need the commonly used utilities | Install the following packages: `pciutils` `vim-minimal` `less` `psmisc` `gnome-keyring`
+**Networking** | If you want networking | Install `qubes-core-agent-networking` `qubes-core-agent-network-manager` `NetworkManager-wifi` `network-manager-applet` `wireless-tools` `dejavu-sans-fonts` `notification-daemon` `gnome-keyring`
```

Comment:
How `dejavu-sans-fonts` is related to networking?

Reply:
FWIW, `dejavu-sans-fonts` has been part of the Fedora minimal template doc. for a [long time](https://web.archive.org/web/20150424102809/https://www.qubes-os.org/doc/Templates/FedoraMinimal/). [This](https://www.qubes-os.org/doc/fedora-minimal-template-customization/#lightweight-packages-recommendations) says it is for "nicer GUI menus" (possibly in Network Manager?), but I don't know if it's still necessary.

Reply:
Both Fedora and CentOS minimal comes with `dejavu-sans-fonts` already installed, so it's not necessary(removed from both templates doc).